src/main/java/io/keyko/monitoring/config/StreamerConfig.java

```diff
@@ -129,4 +151,59 @@ public void setAllTopics(Collection<Object> values) {
     this.topicList = list;
   }
 
+  public String getEtherscanUrl() {
```

Comment:
maybe we can use `lombok` like in the agent to save us of the big POJO classes?

Reply:
could be interesting for next releases